src/StackExchange.Exceptional.Shared/Error.cs

```diff
@@ -189,6 +193,11 @@ private void AddCustomData()
             }
         }
 
+        private void AddExceptionLevel(ExceptionLogLevel level)
+        {
+            ExceptionLevelId = (int)level + 1; // TODO: Get from store, this is hacky
```

Comment:
You could start the enum at 1 with

```
enum Whatever {
  First = 1,
  Second,
  ...
}
```

Reply:
My instinct is that it's better to just query the `ExceptionLevel` table instead of relying on the enum order matching the DB, though maybe for a set as well defined as this it's fine...

Reply:
You can also explicitly add numeric values to every enum entry, so they can be in whatever order you want so long as each has the right value assigned to it.